### PR TITLE
Add first version of the image

### PR DESCRIPTION
**Pull request by guewen**
Docker Hub: https://hub.docker.com/r/camptocamp/odoo-stock-scanner-sentinel/tags/

I built and pushed manually the image on `camptocamp/odoo-stock-scanner-sentinel:9.0-latest`, normally travis should take care of that in the future (I couldn't test it from the pull request though).

### CODE REVIEW

File: shell.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```
Comment (guewen) [inline shell.sh:2]: should we add `set -e` here?

Comment (nbessi) [reply shell.sh:2]: @guewen I do not see the interest. 
We may add a trapp to hide error to customer and log something useful for us

**Review by nbessi — APPROVED** [review]
image tested locally

**Comment by nbessi** [comment]
We can merge

**Pull request merged**